Gemfile

```diff
@@ -31,10 +31,11 @@ gem 'sequel-rails'
 
 gem 'activesupport'
 gem 'base32-crockford'
-gem 'bcrypt-ruby', '~> 3.0.0'
+gem 'bcrypt', '~> 3.1.2'
 gem 'gli', require: false
 gem 'listen'
-gem 'slosilo', '~> 2.1'
+#gem 'slosilo', '~> 2.1'
+gem 'slosilo', github: 'cyberark/slosilo', branch: 'sha256'
```

Comment:
Should publish a new gem version and link to that before merging.

Reply:
Can we get Slosilo released and clean this up before we merge, please?